src/js/css-calc.ts

```diff
@@ -642,44 +683,51 @@ export const sortCalcValues = (values = [], finalize = false) => {
  * @param {string} [opt.format] - output format
  * @returns {?string} - resolved value
  */
-export const serializeCalc = (value, opt = {}) => {
+export const serializeCalc = (
+  value: Array<any>,
+  opt: {
+    format?: string;
+  } = {}
+): string | null => {
   const { format } = opt;
   if (isString(value)) {
     if (!REG_START_MATH_VAR.test(value) || format !== VAL_SPEC) {
       return value;
     }
-    value = value.toLowerCase().trim();
+    value = value.toLowerCase().trim() as never;
   } else {
     throw new TypeError(`${value} is not a string`);
   }
   const cacheKey = `{serializeCalc:${value},opt:${valueToJsonString(opt)}}`;
   if (cachedResults.has(cacheKey)) {
-    return cachedResults.get(cacheKey);
-  }
-  const items = tokenize({ css: value }).map(token => {
-    const [type, value] = token;
-    let res;
-    if (type !== W_SPACE && type !== COMMENT) {
-      res = value;
-    }
-    return res;
-  }).filter(v => v);
-  let startIndex = items.findLastIndex(item => /\($/.test(item));
+    return cachedResults.get(cacheKey) as string;
+  }
+  const items = tokenize({ css: value as never })
+    .map((token) => {
+      const [type, value] = token;
+      let res;
+      if (type !== W_SPACE && type !== COMMENT) {
+        res = value;
+      }
+      return res;
+    })
+    .filter((v: any) => v);
+  let startIndex = items.findLastIndex((item: any) => /\($/.test(item));
```

Comment:
(item: string) =>

Reply:
TypeScript does not allow this type cast because `item` is `string | undefined` so I can either leave it as `any` or change leave it implicitly typed. Which do you prefer?

Reply:
Got it.